Dockerfile.test

```diff
@@ -0,0 +1,21 @@
+FROM python:3.6-stretch
+
+ARG DBOT_ARGS
```

Comment:
Do tests do something different if you pass in "test" vs passing in nothing? If not, this can be removed.

Reply:
Some Avrae files check whether an environment is testing by checking for the presence of `"test" in sys.argv`, which this satisfies - it can probably be hardcoded for the tests, though